gallery/src/components/demo-card.js

```diff
@@ -0,0 +1,71 @@
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import JsYaml from 'js-yaml';
+
+import '../../../src/panels/lovelace/cards/hui-glance-card.js';
```

Comment:
Since the type is configurable, it should be imported in the page that configures the demo card.